package.json

```diff
@@ -83,7 +83,7 @@
     "@tryghost/logging": "2.0.4",
     "@tryghost/magic-link": "1.0.20",
     "@tryghost/member-events": "0.4.0",
-    "@tryghost/members-api": "5.1.1",
+    "@tryghost/members-api": "5.2.0",
```

Comment:
@allouis this also releases your `updateStripeProductNamesFromDefaultProduct` migration - is that cool?

Reply:
Give me a bit of time and I'll merge my bump first if that's cool